cmd/dfdaemon/app/root.go

```diff
@@ -164,7 +164,7 @@ func getConfigFromViper(v *viper.Viper) (*config.Properties, error) {
 }
 
 // decodeWithYAML returns a mapstructure.DecodeHookFunc to decode the given
-// types by unmarshalling from yaml text
+// types by unmarshalling from yaml text.
```

Comment:
If you add . here, you should use uppercase Types

Reply:
// decodeWithYAML returns a mapstructure.DecodeHookFunc to decode the given
// types by unmarshalling from yaml text.   

Isn't this a complete statement?
Really  remove .  ?

Reply:
I am sorry I missed there is one more line above

Reply:
Haha, maybe you need to have a good rest.  Pay attention to your body. 😄

Reply:
hahaha